Packs/CommonPlaybooks/Playbooks/playbook-Containment_Plan_README.md

```diff
@@ -0,0 +1,56 @@
+This playbook handles all the incident containment actions available with Cortex XSIAM, including the following tasks:
+* Isolate endpoint
+* Disable account
+* Quarantine file
+* Block indicators (currently, the playbook supports only hashes)
+* Clear user session (currently, the playbook supports only Okta)
+
+**Note:** The playbook inputs enable manipulating the execution flow; read the input descriptions for details.
```

Comment:
the execution flow or of the execution flow?

Reply:
the execution flow.